packages/odyssey-react/src/components/Table/TableContainer.tsx

```diff
@@ -29,21 +30,21 @@ export interface TableContainerProps
   /**
    * The visible heading for the table
    */
-  title: ReactNode;
+  heading: ReactNode;
 }
 
 export const TableContainer = withTheme(
-  () => ({}),
+  theme,
   styles
 )(
   forwardRef<HTMLElement, TableContainerProps>((props, ref) => {
-    const { children, title, ...rest } = props;
+    const { children, heading, ...rest } = props;
 
     const omitProps = useOmit(rest);
 
     return (
       <Box as="figure" {...omitProps} ref={ref} className={styles.root}>
-        <figcaption className={styles.title}>{title}</figcaption>
+        <figcaption className={styles.heading}>{heading}</figcaption>
```

Comment:
wondering why this isn't using `<Heading />` here.....

Reply:
@edburyenegren-okta 

> 1. Heading doesn't offer the correct element for captioning a figure

hmmm we should consider renaming the prop `heading` here. Something like `caption` would be more accurate and won't carry the connotations of heading DOM elements (h1, h2 ,h3) or our `<Heading>` react component! That helps remove ambiguity. If I pass a string to a `heading` prop my assumption is that I get a heading element in the DOM.

> Composition doesn't allow for theming yet

I'm not sure I follow. Composition does allow for theming, if we imagine that this **was** using `<Heading />` via composition then we would target the styles to change like so:

```
theme={{
   [Heading.theme]: { ... }
}}
```

Reply:
If this works the same was as theming Box, theming `Heading` themes the component globally, not Table's Heading only. Given that products utilize different headings at different levels, that won't be a workable solve for Table.